diff -r -U3 ../../../util/libav-0.9/AVUtil/v52/libavutil_h.jl v52/libavutil_h.jl
--- ../../../util/libav-0.9/AVUtil/v52/libavutil_h.jl	2014-07-22 00:47:48.214270830 -0700
+++ v52/libavutil_h.jl	2014-07-22 02:38:25.910151961 -0700
@@ -394,11 +394,16 @@
 const FF_LAMBDA_MAX = 256 * 128 - 1
 const FF_QUALITY_SCALE = FF_LAMBDA_SCALE
 
-# Skipping MacroDefinition: AV_NOPTS_VALUE INT64_C ( 0x8000000000000000 )
+const AV_NOPTS_VALUE = reinterpret(Int64, 0x8000000000000000)
 
 const AV_TIME_BASE = 1000000
 
-# Skipping MacroDefinition: AV_TIME_BASE_Q ( AVRational ) { 1 , AV_TIME_BASE }
+immutable AVRational
+    num::Cint
+    den::Cint
+end
+
+const AV_TIME_BASE_Q = AVRational( 1 , AV_TIME_BASE )
 
 # begin enum AVMediaType
 typealias AVMediaType Cint
@@ -591,42 +596,6 @@
     comp::Array_4_AVComponentDescriptor
 end
 
-# Skipping MacroDefinition: AV_PIX_FMT_NE ( be , le ) AV_PIX_FMT_ ## le
-# Skipping MacroDefinition: AV_PIX_FMT_RGB32 AV_PIX_FMT_NE ( ARGB , BGRA )
-# Skipping MacroDefinition: AV_PIX_FMT_RGB32_1 AV_PIX_FMT_NE ( RGBA , ABGR )
-# Skipping MacroDefinition: AV_PIX_FMT_BGR32 AV_PIX_FMT_NE ( ABGR , RGBA )
-# Skipping MacroDefinition: AV_PIX_FMT_BGR32_1 AV_PIX_FMT_NE ( BGRA , ARGB )
-# Skipping MacroDefinition: AV_PIX_FMT_GRAY16 AV_PIX_FMT_NE ( GRAY16BE , GRAY16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_RGB48 AV_PIX_FMT_NE ( RGB48BE , RGB48LE )
-# Skipping MacroDefinition: AV_PIX_FMT_RGB565 AV_PIX_FMT_NE ( RGB565BE , RGB565LE )
-# Skipping MacroDefinition: AV_PIX_FMT_RGB555 AV_PIX_FMT_NE ( RGB555BE , RGB555LE )
-# Skipping MacroDefinition: AV_PIX_FMT_RGB444 AV_PIX_FMT_NE ( RGB444BE , RGB444LE )
-# Skipping MacroDefinition: AV_PIX_FMT_BGR48 AV_PIX_FMT_NE ( BGR48BE , BGR48LE )
-# Skipping MacroDefinition: AV_PIX_FMT_BGR565 AV_PIX_FMT_NE ( BGR565BE , BGR565LE )
-# Skipping MacroDefinition: AV_PIX_FMT_BGR555 AV_PIX_FMT_NE ( BGR555BE , BGR555LE )
-# Skipping MacroDefinition: AV_PIX_FMT_BGR444 AV_PIX_FMT_NE ( BGR444BE , BGR444LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV420P9 AV_PIX_FMT_NE ( YUV420P9BE , YUV420P9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV422P9 AV_PIX_FMT_NE ( YUV422P9BE , YUV422P9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV444P9 AV_PIX_FMT_NE ( YUV444P9BE , YUV444P9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV420P10 AV_PIX_FMT_NE ( YUV420P10BE , YUV420P10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV422P10 AV_PIX_FMT_NE ( YUV422P10BE , YUV422P10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV444P10 AV_PIX_FMT_NE ( YUV444P10BE , YUV444P10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV420P16 AV_PIX_FMT_NE ( YUV420P16BE , YUV420P16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV422P16 AV_PIX_FMT_NE ( YUV422P16BE , YUV422P16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUV444P16 AV_PIX_FMT_NE ( YUV444P16BE , YUV444P16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_GBRP9 AV_PIX_FMT_NE ( GBRP9BE , GBRP9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_GBRP10 AV_PIX_FMT_NE ( GBRP10BE , GBRP10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_GBRP16 AV_PIX_FMT_NE ( GBRP16BE , GBRP16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA420P9 AV_PIX_FMT_NE ( YUVA420P9BE , YUVA420P9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA422P9 AV_PIX_FMT_NE ( YUVA422P9BE , YUVA422P9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA444P9 AV_PIX_FMT_NE ( YUVA444P9BE , YUVA444P9LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA420P10 AV_PIX_FMT_NE ( YUVA420P10BE , YUVA420P10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA422P10 AV_PIX_FMT_NE ( YUVA422P10BE , YUVA422P10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA444P10 AV_PIX_FMT_NE ( YUVA444P10BE , YUVA444P10LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA420P16 AV_PIX_FMT_NE ( YUVA420P16BE , YUVA420P16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA422P16 AV_PIX_FMT_NE ( YUVA422P16BE , YUVA422P16LE )
-# Skipping MacroDefinition: AV_PIX_FMT_YUVA444P16 AV_PIX_FMT_NE ( YUVA444P16BE , YUVA444P16LE )
-
 # begin enum AVPixelFormat
 typealias AVPixelFormat Cint
 const AV_PIX_FMT_NONE = int32(-1)
@@ -835,7 +804,36 @@
 
 const PixelFormat = AVPixelFormat
 
-# Skipping MacroDefinition: PIX_FMT_NE ( be , le ) AV_PIX_FMT_NE ( be , le )
+
+macro AV_PIX_FMT_NE(be,le)
+    symbol("AV_PIX_FMT_",le)
+end
+ 
+const AV_PIX_FMT_RGB32 = @AV_PIX_FMT_NE(ARGB,BGRA)
+const AV_PIX_FMT_RGB32_1 = @AV_PIX_FMT_NE(RGBA,ABGR)
+const AV_PIX_FMT_BGR32 = @AV_PIX_FMT_NE(ABGR,RGBA)
+const AV_PIX_FMT_BGR32_1 = @AV_PIX_FMT_NE(BGRA,ARGB)
+const AV_PIX_FMT_GRAY16 = @AV_PIX_FMT_NE(GRAY16BE,GRAY16LE)
+const AV_PIX_FMT_RGB48 = @AV_PIX_FMT_NE(RGB48BE,RGB48LE)
+const AV_PIX_FMT_RGB565 = @AV_PIX_FMT_NE(RGB565BE,RGB565LE)
+const AV_PIX_FMT_RGB555 = @AV_PIX_FMT_NE(RGB555BE,RGB555LE)
+const AV_PIX_FMT_RGB444 = @AV_PIX_FMT_NE(RGB444BE,RGB444LE)
+const AV_PIX_FMT_BGR48 = @AV_PIX_FMT_NE(BGR48BE,BGR48LE)
+const AV_PIX_FMT_BGR565 = @AV_PIX_FMT_NE(BGR565BE,BGR565LE)
+const AV_PIX_FMT_BGR555 = @AV_PIX_FMT_NE(BGR555BE,BGR555LE)
+const AV_PIX_FMT_BGR444 = @AV_PIX_FMT_NE(BGR444BE,BGR444LE)
+const AV_PIX_FMT_YUV420P9 = @AV_PIX_FMT_NE(YUV420P9BE,YUV420P9LE)
+const AV_PIX_FMT_YUV422P9 = @AV_PIX_FMT_NE(YUV422P9BE,YUV422P9LE)
+const AV_PIX_FMT_YUV444P9 = @AV_PIX_FMT_NE(YUV444P9BE,YUV444P9LE)
+const AV_PIX_FMT_YUV420P10 = @AV_PIX_FMT_NE(YUV420P10BE,YUV420P10LE)
+const AV_PIX_FMT_YUV422P10 = @AV_PIX_FMT_NE(YUV422P10BE,YUV422P10LE)
+const AV_PIX_FMT_YUV444P10 = @AV_PIX_FMT_NE(YUV444P10BE,YUV444P10LE)
+const AV_PIX_FMT_YUV420P16 = @AV_PIX_FMT_NE(YUV420P16BE,YUV420P16LE)
+const AV_PIX_FMT_YUV422P16 = @AV_PIX_FMT_NE(YUV422P16BE,YUV422P16LE)
+const AV_PIX_FMT_YUV444P16 = @AV_PIX_FMT_NE(YUV444P16BE,YUV444P16LE)
+const AV_PIX_FMT_GBRP9 = @AV_PIX_FMT_NE(GBRP9BE,GBRP9LE)
+const AV_PIX_FMT_GBRP10 = @AV_PIX_FMT_NE(GBRP10BE,GBRP10LE)
+const AV_PIX_FMT_GBRP16 = @AV_PIX_FMT_NE(GBRP16BE,GBRP16LE)
 
 const PIX_FMT_RGB32 = AV_PIX_FMT_RGB32
 const PIX_FMT_RGB32_1 = AV_PIX_FMT_RGB32_1
@@ -863,11 +861,6 @@
 const PIX_FMT_GBRP10 = AV_PIX_FMT_GBRP10
 const PIX_FMT_GBRP16 = AV_PIX_FMT_GBRP16
 
-immutable AVRational
-    num::Cint
-    den::Cint
-end
-
 # begin enum AVSampleFormat
 typealias AVSampleFormat Cint
 const AV_SAMPLE_FMT_NONE = int32(-1)
Only in v52: libavutil_h.jl.orig
Only in v52: libavutil_h.jl.rej
diff -r -U3 ../../../util/libav-0.9/AVUtil/v52/LIBAVUTIL.jl v52/LIBAVUTIL.jl
--- ../../../util/libav-0.9/AVUtil/v52/LIBAVUTIL.jl	2014-07-22 00:47:48.210270830 -0700
+++ v52/LIBAVUTIL.jl	2014-07-22 02:31:16.958159643 -0700
@@ -10,7 +10,7 @@
 #include("log.jl")
 #include("mem.jl")
 #include("opt.jl")
-#include("pixdesc.jl")
+include("pixdesc.jl")
 #include("rational.jl")
 #include("samplefmt.jl")
 #include("xtea.jl")
Only in v52: LIBAVUTIL.jl.orig
